asm/macros/movement.inc

```diff
@@ -14,10 +14,10 @@
 	create_movement face_up_fast
 	create_movement face_left_fast
 	create_movement face_right_fast
-	create_movement walk_slowest_down
-	create_movement walk_slowest_up
-	create_movement walk_slowest_left
-	create_movement walk_slowest_right
+	create_movement walk_slower_down
+	create_movement walk_slower_up
+	create_movement walk_slower_left
+	create_movement walk_slower_right
```

Comment:
Should also create movement macros for the newly named movements (even though they're not used by vanilla scripts)